lib/src/evaluation/evaluator_manager.dart

```diff
@@ -62,15 +46,13 @@ class EvaluatorManager {
 
   /// Spawns a background Isolate for the manager logic.
   static Future<EvaluatorManager> spawn() async {
-    // Create a receive port and add its initial message handler
     final initPort = RawReceivePort();
     final connection = Completer<(ReceivePort, SendPort)>.sync();
     initPort.handler = (initialMessage) {
       final commandPort = initialMessage as SendPort;
       connection.complete((ReceivePort.fromRawReceivePort(initPort), commandPort));
     };
 
-    // Spawn the isolate.
     try {
       await Isolate.spawn(evaluatorManagerIsolateEntrypoint, (initPort.sendPort));
     } on Object {
```

Comment:
nit: `finally` for closing the port.
```suggestion
    } finally {
```

Reply:
This is incorrect. 
1. The port will be closed always, which is not what we want
2. Rethrow in `finally` block will trigger compile time error. 

Existing code is correct and is what we want. `Object` is a catch all for all errors.